packages/evm/contracts/budgets/TransparentBudget.sol

```diff
@@ -145,6 +148,21 @@ contract TransparentBudget is ATransparentBudget, ReentrancyGuard {
         revert BoostError.NotImplemented();
     }
 
+    function clawbackFromTarget(address target, bytes calldata data_, uint256 boostId, uint256 incentiveId)
+        external
+        virtual
+        override
+        returns (uint256, address)
+    {
+        BoostLib.Boost memory boost = BoostCore(target).getBoost(boostId);
+        if (msg.sender != boost.owner) revert BoostError.Unauthorized();
+        AIncentive.ClawbackPayload memory payload =
+            AIncentive.ClawbackPayload({target: address(msg.sender), data: data_});
```

Comment:
It would be nice to provide an option to clawback the funds to an arbitrary address but I'm not seeing a great way to do that without either modifying the interface or changing assumptions about the shape of the data payload. Future problem and/or we can support if there's demand.

Reply:
yes my thinking as well